.vscode/settings.json

```diff
@@ -6,5 +6,8 @@
     "**/CVS": true,
     "**/.DS_Store": true,
     "**/.rpt2_cache": true
-  }
+  },
```

Comment:
tries to auto fix tslint errors on save - I like it but can rm if annoying

Reply:
I was thinking about it. A good idea in general but there may be some edge cases